cmd/commands/service/command.go

```diff
@@ -131,13 +143,18 @@ func (sc *serviceCommand) Run(ctx *cli.Context) (err error) {
 	return <-sc.errorChannel
 }
 
-func (sc *serviceCommand) unlockIdentity(identityOptions service.OptionsIdentity) (*identity.Identity, error) {
-	id, err := sc.tequilapi.CurrentIdentity(identityOptions.Identity, identityOptions.Passphrase)
-	if err != nil {
-		return nil, err
-	}
+const retryRate = 10 * time.Second
```

Comment:
Can we put the const on top of the file?

Reply:
Having it next to the code using it is easier to understand/discover, but I can move it.

Reply:
Moved it to the inside of the function. There's no need to expose it to package.